src/components/common/transactions-list/transactions-list.component.coffee

```diff
@@ -54,6 +56,38 @@ module.component('transactionsList', {
       trx.datePicker.date = expDate
       ctrl.onUpdateExpectedDate({ trxId: trx.id, date: trx.datePicker.date })
 
+    ctrl.canCreateSchedulableTransaction = (trx) ->
+      return trx.status != 'FORECAST' && !trx.recurring
+
+    ctrl.canDeleteSchedulableTransaction = (trx) ->
+      return (trx.status == 'FORECAST' && (trx.recurring || trx.recurring_parent)) || (trx.status != 'FORECAST' && trx.recurring)
+
+    ctrl.deleteSchedule =
+      args: {}
```

Comment:
I dont think this is used?

Reply:
Yeah, it's used to display the dialog for the user confirm or not if he wants to delete the schedule for the transaction.